substratevm/src/com.oracle.svm.core.graal.llvm/src/com/oracle/svm/core/graal/llvm/util/LLVMIRBuilder.java

```diff
@@ -1073,6 +1074,45 @@ public LLVMValueRef buildBswap(LLVMValueRef a) {
         return buildIntrinsicOp("bswap", a);
     }
 
+    // LLVM fptosi instruction returns poison if the input is NaN or outside the integer range.
+    // However, LLVM llvm.fptosi.sat.* intrinsic functions follow the Java semantics.
+    public LLVMValueRef buildSaturatingFloatingPointToInteger(FloatConvert op, LLVMValueRef a) {
+        final String intrinsicName;
+        final LLVMTypeRef retType;
+        final LLVMTypeRef[] argsType = new LLVMTypeRef[1];
```

Comment:
`functionType` is variadic, no need to create an array here.

Reply:
Done.